dwave/optimization/mathematical.py

```diff
@@ -519,6 +521,57 @@ def multiply(x1: ArraySymbol, x2: ArraySymbol, *xi: ArraySymbol,
     raise RuntimeError("implementated by the op() decorator")
 
 
+def put(array: ArraySymbol, indices: ArraySymbol, values: ArraySymbol) -> Put:
+    r"""Replace the specified elements in an array with given values.
+
```

Comment:
Probably worth having a note here about expected behavior in the case of duplicate values, even though numpy does not specify this in their docs.

Reply:
Should be fixed by https://github.com/dwavesystems/dwave-optimization/pull/201/commits/88134d6571a8e5e8d8161701b98647f779cb8ea3